viewing_party/party.py

```diff
@@ -1,23 +1,213 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+from symbol import single_input
+
+
+def create_movie(title, genre, rating): #Wave 1, step 1 done
+    movie_dict={}
+    if title is None or genre is None or rating is None:
+        return None

```

Comment:
Think about putting this at the top of the function first. It saves us from creating space in memory, defining variables, etc. that we are simply going to throw away because the parameters didn't meet our rules, ie, `movie_dict`